cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py

```diff
@@ -42,11 +42,66 @@ def _is_segm_id_test(self, param, name):
         return param in PARAMS_NAMES_TO_OMIT and (
             name in NET_NAMES_TO_OMIT or name in SUBNET_NAMES_TO_OMIT)
 
+    def get_attribute_val(self, attr_list, resource_name, parameter):
+        temp = {}
+        temp[resource_name] = {}
+        temp[resource_name][parameter] = getattr(attr_list, parameter)
+        temp[resource_name]["name"] = getattr(attr_list, "name").lower()
+        return temp[resource_name][parameter]
```

Comment:
The "name" parameter for the resource (in this case the user or the tenant) will be returned in lower case. For other parameters, the case is not supposed to be changed.  Thats why I wrote two lines below. It will update the value of "name" parameter to lower case.

temp[resource_name][parameter] = getattr(attr_list, parameter)
temp[resource_name]["name"] = getattr(attr_list, "name").lower()